clouddq/classes/rule_type.py

```diff
@@ -112,13 +132,13 @@ def to_sql_custom_sql_statement(params: dict) -> Template:
                 raise ValueError(
                     f"RuleType: {RuleType.CUSTOM_SQL_STATEMENT} "
                     f"custom_sql_arguments '${argument}' not found in "
-                    f"custom_sql_statement:\n {custom_sql_statement}"
+                    f"custom_sql_statement:\n {pformat(custom_sql_statement)}"
                 )
-            if not params.get("rule_binding_arguments", {}).get(argument, None):
+            if params.get("rule_binding_arguments", {}).get(argument, None) is None:
```

Comment:
Very helpful bugfix (I came across this issue recently). Shouldn't it have a unit test case, e.g. with 
param_value=0, param_value="", param_value= 
?

Reply:
see above :)